spec/ddtrace/configuration/settings_spec.rb

```diff
@@ -476,7 +476,7 @@
       subject(:rate_limit) { settings.sampling.rate_limit }
 
       context 'default' do
-        it { is_expected.to be 100 }
+        it { is_expected.to eq(100) }
```

Comment:
`be` performs identity comparison: same object.
`eq` performs equivalence comparison: same value.

There are many instances of `be(Numeric)` in our tests that were working because of Ruby MRI reuses the same object instance for small numbers, but JRuby doesn't in all cases.